crossdock/jaeger-docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '2'
```

Comment:
Why use a new file?

Reply:
because I only want to affect crossdock.

Reply:
I see your point. I like the approach.